packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-drilldown.vue

```diff
@@ -458,6 +458,7 @@ useClientEvent(ClientEventType.TaskGollmConfigureFromDataset, configModelEventHa
 
 const selectedOutputId = ref<string>('');
 const selectedConfigId = computed(() => props.node.outputs.find((o) => o.id === selectedOutputId.value)?.value?.[0]);
+const selectedModelConfig = ref<ModelConfiguration | null>(null);
```

Comment:
I think this can be a regular variable instead of a `ref`. 

As an aside, after working in here myself I noticed that the `originalConfig` is retrieved in `resetConfiguration()`. Maybe you can just use the `selectedModelConfig` you made here there to avoid the pointless `find()`...maybe renaming it to `originalConfig` would make more sense too.